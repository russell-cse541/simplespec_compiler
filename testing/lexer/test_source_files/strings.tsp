"hello this is an ascii string"
// 'hello this is a unicode string \u018e and % \xc6\x8e string \xc3\xb1 hopefully!'
// "hello this is an ascii string"
'hello this is a unicode string \u018e and % \xc6\x8e string \xc3\xb1 hopefully!'