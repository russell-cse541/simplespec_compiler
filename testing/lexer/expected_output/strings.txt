
"hello this is an ascii string"   TagNum: 309    Line: 1     Col: 1          
'hello this is a unicode string \u018e and % \xc6\x8e string \xc3\xb1 hopefully!'   TagNum: 310    Line: 4     Col: 1          
There were no unrecognized tokens
Token Count : 2
